Modules/class.js

```diff
@@ -51,7 +51,7 @@ class BookList {
   displayBook(book) {
     const bookList = document.getElementById('booklist');
     var table = document.createElement("table");
-    table.classList.add("table", "table-striped", );
+    table.classList.add("table", "table-striped" );
        table.innerHTML = `
       <tr>
       <td>
```

Comment:
- [ ] Kindly remove the `CurrentDate` function and the `window.onload` call from this file since you are using the `DateTime` object from Luxon Library. This would keep your code **_DRY (Don't Repeat Yourself)_**. Kindly make this change. 👍 😎